relation/src/gadgets/emulated.rs

```diff
@@ -553,6 +567,17 @@ impl<F: PrimeField> PlonkCircuit<F> {
         Ok(c)
     }
 
+    /// Return an [`EmulatedVariable`] which equals to a+b.
```

Comment:
a+b -> a-b, also mention b is a constant.

Reply:
Done in [f4af91b](https://github.com/EspressoSystems/jellyfish/pull/326/commits/f4af91b6934fba66b14a7e24e8e28f673ebc67ba)